src/cli/rdb-cli.c

```diff
@@ -372,9 +377,13 @@ int main(int argc, char **argv)
         return RDB_ERR_GENERAL;
     }
 
-    if ((logfile = fopen(options.logfilePath, "w")) == NULL) {
-        printf("Error opening log file for writing: %s \n", options.logfilePath);
-        return RDB_ERR_GENERAL;
+    if (strcmp(options.logfilePath, "-") == 0) {
+        logfile = stdout;
+    } else {
+        if ((logfile = fopen(options.logfilePath, "w")) == NULL) {
+            printf("Error opening log file for writing: %s \n", options.logfilePath);
```

Comment:
We can also print failure reason with `strerror(errno)`.

Reply:
Good idea. thanks.